deployment.md

```diff
@@ -0,0 +1,107 @@
+## Deployment
+Configuration management and deployment for every data.gov component follows the pattern described unless otherwise specified.
+
+Figure 10-1 Deployment Diagram
+
```

Comment:
Can you add the preview image here?

Reply:
![image](https://user-images.githubusercontent.com/75858167/103218526-3644f200-48e9-11eb-8a01-0c86564c3ac1.png)

Reply:
![image](https://user-images.githubusercontent.com/75858167/103218623-82903200-48e9-11eb-925c-5f0847aa0687.png)

Reply:
Need help on this one